modules/websocket_bridge/include/hephaestus/utils/protobuf_serdes.h

```diff
@@ -0,0 +1,139 @@
+//=================================================================================================
```

Comment:
this file paht should be
`modules/websocket_bridge/include/hephaestus/websocket_bridge/utils/protobuf_serdes.h`